avocado/utils/cpu.py

```diff
@@ -188,3 +189,56 @@ def offline(cpu):
     if _get_cpu_status(cpu):
         return 1
     return 0
+
+
+def get_cpu_idle():
+    """
+    Get current cpu idle values
+    """
+    cpus_list = cpu_online_list()
+    states = range(len(glob.glob("/sys/devices/system/cpu/cpu0/cpuidle/state*")))
```

Comment:
There is probably a typo and you meant `cpu/cpu*`, didn't you?

Reply:
Oh, actually you didn't... Sorry, tired eyes...

Reply:
Btw have you considered raising an exception in case cpuidle is not supported? I think I'd prefer that, anyway the current code works well without it as well so I'm fine with both. Just asking whether you considered such scenario.

Reply:
eg. arm machine ;-)